google/gax/auth.py

```diff
@@ -34,7 +34,7 @@
 from oauth2client import client as auth_client
 
 
-def make_auth_func(scopes):
+def make_auth_func(scopes, agent):
```

Comment:
We should not do this.

Even if this works now, we cannot rely on it working continuously as the grpc core aims to restrict the usage of the auth_func to valid authorization headers